cibyl/models/ci/base/system.py

```diff
@@ -190,6 +192,7 @@ def __init__(self,
             top_level_model=Job,
             sources=sources,
             enabled=enabled,
+            jobs=jobs  # pass jobs to parent init so it's not overriden
```

Comment:
No, since jobs_scope is not in the API does not need to go through the Model __init__. It's stored as a simple string, not an Argument. Now that you mention it, I'll add a comment for it since it's not obvious